tests/Unit/FileTest.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+namespace MyParcelCom\ApiSdk\Tests\Unit;
+
+use MyParcelCom\ApiSdk\Resources\File;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\StreamInterface;
+
+class FileTest extends TestCase
+{
+    /** @var array */
+    private $pngFormat;
+    /** @var array */
+    private $jpegFormat;
+    /** @var array */
+    private $pdfFormat;
+    /** @var StreamInterface */
+    private $stream;
+    /** @var string */
+    private $base64data;
+    /** @var string */
+    private $path;
+
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->pngFormat = [
+            'mime_type' => 'image/png',
+            'extension' => 'png',
+        ];
+
+        $this->jpegFormat = [
+            'mime_type' => 'image/jpeg',
+            'extension' => 'jpg',
+        ];
+
+        $this->pdfFormat = [
+            'mime_type' => 'application/pdf',
+            'extension' => 'pdf',
+        ];
+
+        $this->stream = $this->createMock(StreamInterface::class);
+        $this->stream
+            ->method('getContents')
+            ->willReturn('This is some test data.');
```

Comment:
If you put this string in a constant, it will be clearer that you're using the same data everywhere.

You should also use different values for the stream, base64 data and path. Then you can check for the correct value when setting data through a stream, base64 data or a path.